src/App.tsx

```diff
@@ -1,9 +1,19 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  function searchInput(input: string) {
+    return input.trim().toLowerCase().includes(query.toLowerCase());
+  }
+
+  const searchResult = moviesFromServer.filter(
```

Comment:
may be better to use given variable 'visibleMovies'?

Reply:
agree